### ISSUE TITLE
-e arg isn't working in Homebrew version (but works fine when compiled from source)

### ISSUE THREAD

**Issue by luckman212** [tenox7/ttyplot#49]
Hey, first: amazing work on this tool! I'm using it to visualize memory pressure over time on my Mac.

v1.4 from Homebrew seems to have a problem with the `-e` option -- e.g. this will fail (just prints the helptext and then pauses for 1 second, then exits):
```bash
while [0]; do memory_pressure | awk 'END {print $5/1}'; sleep 1; done | ttyplot -e+ -cx -m100 -t%free_mem -u%
```
but, cloning `master` and compiling with `make` outputs a version where the exact same command works fine (also says 1.4 but obviously something's different).

macOS 10.15.5

Any idea?

**Comment by tenox7** [comment]
very interesting, let me investigate

**Comment by hartwork** [comment]
Turns out that:
- I can reproduce the problem for Git tag `1.4` locally
- The problem is that the `optstring` passed to `getopt` is missing `e`
- The problem has been fixed by commit d6a5ec90c742c4765ca16eed814fa61dadb68cd5 already
- The first release including the fix is 1.5(.0)
- Issue #40 already reported the same problem but was already closed as fixed (but not released) when issue #49 here was opened

I'll close as fixed, please re-open as needed.
